src/screens/SendSheet.js

```diff
@@ -153,7 +153,7 @@ const SendSheet = ({
         });
       } else {
         setSelected(newSelected);
-        sendUpdateAssetAmount(amountDetails.assetAmount);
+        sendUpdateAssetAmount('');
```

Comment:
@wojtus7  would you mind explaining the logic behind this change?

Reply:
@brunobar79 Couple of lines above it is set to when selecting NFT: 

```setAmountDetails({
          assetAmount: '1',
          isSufficientBalance: true,
          nativeAmount: '0',
        });```

Then there was this bug - when you selected NFT first then go back to list and select some coin without quitting the bottom sheet the script was taking this old NFT `1` value and that was initial value for coins for no reason.
I send an empty string because that way it was initiated in line 83 to reset this NFT state.

Reply:
That makes sense. Thanks for the explanation!